stacks/end-to-end-security/hive-metastore.yaml

```
@@ -7,6 +7,10 @@ spec:
   image:
     productVersion: 4.0.0
   clusterConfig:
+    authorization:
+      opa:
+        configMapName: opa
+        package: hms
```

No strong opinion, but WDYT of calling the package the same as the stacklet?
We could technically have multiple HMS, which need different rules
```suggestion
        package: hive-iceberg
```